main.js

```diff
@@ -0,0 +1,124 @@
+
+// ******************************* VARIABLES ******************************* //
+const currentTime = moment().unix();
+const weatherAPI = '5a6305ded7306f7187c4e5b53e78ac91';
+
+// ******************************* FUNCTIONS ******************************* //
+// Generates list of countries for select dropdown
+const renderCountriesList = () => {
+  const $countrySelect = $(`.country-select`);
+  const template = Handlebars.compile($(`#option-template`).html());
+
+  countryList.forEach(country => {
+    $countrySelect.append(template(country));
+  });
+};
+
+// Returns object of temperature in imperial (F) and metric (C) units
+const calculateTemps = temp => {
```

Comment:
Hey Edward, this approach is fine but a little unnecessary. The open weather api end point accepts a  `units=imperial` parameter, and by default its set to celsius.